docs/uninstall.md

```diff
@@ -54,11 +55,10 @@ ofn uninstall --all --wait
 
 ### Uninstall a specific version of OpenFunction
 
-> Default to v0.4.0 if no version specified
+> Default to the version of the OpenFunction exists
```

Comment:
Default to the version of the OpenFunction exists 
=>
Default to the version of the OpenFunction currently installed